### PR TITLE
Add EC/TC/OC duties and update TC/OC list

### PR DESCRIPTION
**Pull request by kyordhel**
As presented in #429, this PR adds the EC/TC/OC duties

### CODE REVIEW

**Comment by kyordhel** [comment]
@airglow Agree.

The PR was on request. Any suggestions on where to hang this info?

File: introduction/Organization.tex [hunk 1]
```diff
@@ -6,39 +6,100 @@
...
+% %% %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
 \subsection{Technical Committee --- tc@robocupathome.org}
 \label{sec:tc}
-The \iaterm{Technical Committee}{TC} is responsible for the rules of each league. Members of the RoboCup@Home Technical Committee for \YEAR:
+The \iaterm{Technical Committee}{TC} is responsible for the rules of each league. Its responsibilities include:
+
+\begin{enumerate}
+  \item During preparation time
+  \begin{itemize}
+    \item Write and buff the rules in the rulebook
```
Comment (LoyVanBeek) [inline introduction/Organization.tex]: buff --> improve

File: introduction/Organization.tex [hunk 2]
```diff
@@ -6,39 +6,100 @@
...
+    \item Design and write new tests (when applicable)
+    \item Evaluate previous performance and update tests accordingly
+  \end{itemize}
+  \item During the competition
+  \begin{itemize}
+    \item Direct Q\&A sessions during Team Leader meetings
+    \item Renounce to their teams and serves as referees (see \refsec{rule:referees})
```
Comment (LoyVanBeek) [inline introduction/Organization.tex]: During competitions :-). 

TC members must not misuse their TC role to benefit their team (should be obvious though)

File: introduction/Organization.tex [hunk 3]
```diff
@@ -6,39 +6,100 @@
...
+    \item Take decisions about partial scores and penalties
+  \end{itemize}
+\end{enumerate}
+
+The \iaterm{Technical Committee}{TC} is renewed every year, being its members elected by the participating teams. However, elected members still have to earn their place in the TC during the running year by actively contributing with the next-year's rulebook creation. The Executive Committee reserves to itself the legitimate right of replacing inactive TC members with more enthusiastic and participative people from the league who has been actively contributing during the year. It is advised that candidates work with the \iaterm{Organizing Committee}{OC} for one year before trying to join the Technical Committee.
```
Comment (LoyVanBeek) [inline introduction/Organization.tex]: OC before TC is new to me. Perhaps Contributing to the rulebook is also a precursor to TC-ing?

File: introduction/Organization.tex [hunk 4]
```diff
@@ -6,39 +6,100 @@
...
+    \item Guard the arena access during tests
+    \item Organize queues during tests
+    \item Retrieve signed scoresheets and capture scores
+    \item Compute and publish scores
+    \item Perform OC tasks stated in the rulebook
+    \item Any other required logistic activities
+  \end{itemize}
+\end{enumerate}
+
+The \iaterm{Organizing Committee}{OC} activities are directed by the \iterm{OC Chair}, and its members are self-elected by volunteering. Participants can join the Organizing Committee anytime just by committing to the OC Chair, however, only participants with at least two years of experience in RoboCup@Home can take par in the Qualification Procedure (see \refsec{rule:participation}).
```
Comment (LoyVanBeek) [inline introduction/Organization.tex]: par --> part

**Comment by LoyVanBeek** [comment]
@airglow I'm in dubio. having this clear is important to me, but perhaps it is something more suited for an appendix or even wiki page?

File: introduction/Organization.tex (hunk 3 above)
Comment (kyordhel) [reply introduction/Organization.tex]: Advised but not mandatory. As of 2009~2012 was mentioned by Execs right before the elections.

**Comment by kyordhel** [comment]
@LoyVanBeek Thanks for the review. 
However, this PR shouldn't be merged until we all agree this info is necessary in the Rulebook. I agree with @airglow in that we should start trimming

**Comment by semeyerz** [comment]
#429 "Write logs of teamleader-meetings and make publicly accessible" is not really addressed with this PR.
Can you please add a task to the OC that they select someone for writing a log?

File: introduction/Organization.tex [hunk 5]
```diff
@@ -6,39 +6,103 @@
...
+The \iaterm{Organizing Committee}{OC} activities are directed by the \iterm{OC Chair}, and its members are self-elected by volunteering. Participants can join the Organizing Committee anytime just by committing to the OC Chair, however, only participants with at least two years of experience in RoboCup@Home can take part in the Qualification Procedure (see \refsec{rule:participation}).
+
+\subsubsection{Members of the RoboCup@Home Organizing Committee for \YEAR}
 
 \begin{itemize}
-  \item Mauricio Matamoros (Universidad Nacional Autonoma de Mexico, Mexico)
-  \item Francisco Javier Rodriguez Lera (University of Luxembourg, Luxembourg)
-  \item Raphael Memmesheimer (University of Koblenz, Germany)
+  \item[Chair] Mauricio Matamoros (University of Koblenz-Landau, Germany)
   \item Alexander William Moriarty (TBA, Bonn-Rhein-Sieg University; Germany)
```
[the remark this answers is not in the document]
Comment (kyordhel) [reply introduction/Organization.tex]: K, minor fix! It's done to USA. You are gringo now, amigo.

File: introduction/Organization.tex [hunk 6]
```diff
@@ -6,39 +6,103 @@
...
+
+The \iaterm{Technical Committee}{TC} is renewed every year, being its members elected by the participating teams. However, elected members still have to earn their place in the TC during the running year by actively contributing with the next-year's rulebook creation.
+The Executive Committee reserves to itself the legitimate right of replacing inactive TC members with more enthusiastic and participative people from the league who has been actively contributing during the year.
+It is advised that candidates contribute with the rulebook or work with the \iaterm{Organizing Committee}{OC} for one year before joining the Technical Committee.
+
+The \iaterm{Technical Committee}{TC} does not have a Chair, but its actions are directed and supervised by the Executive Committee. All members of the EC are considered to be part also of the TC.
+
+\subsubsection{Members of the RoboCup@Home Technical Committee for \YEAR}:
 \begin{itemize}
-  \item Loy Van Beek (Eindhoven University of Technology, The Netherlands)
```
[the remark this answers is not in the document]
Comment (kyordhel) [reply introduction/Organization.tex]: You are TU/e member and participate/contribute with the league as such.

BTW, this patch only adds TC duties, it doesn't change the members of each committee.

**Comment by kyordhel** [comment]
@semeyerz Record added. However, as with taking times, I wouldn't be that confident of having someone for that task.

File: introduction/Organization.tex (hunk 6 above)
[the remark this answers is not in the document]
Comment (kyordhel) [reply introduction/Organization.tex]: Loy, you just volunteered yourself for the OC.

Thank you!!!

**Comment by kyordhel** [comment]
@LoyVanBeek @moriarty Please review changes
This PR won't be merged as @airglow disagrees with it. I'll open a PR with name fixes only.

**Comment by LoyVanBeek** [comment]
I opened https://github.com/RoboCupAtHome/AtHomeCommunityWiki/wiki/Committee-responsibilities in anticipation of the names+affiliation PR. 

@semeyerz https://github.com/RoboCupAtHome/AtHomeCommunityWiki/wiki/Committee-responsibilities/_compare/f8c8bb33f9cf0060c18d3f7637ceeb76c4493118...f6230d1a012ff1d8f12b52b0ad70b2f0d0600fba OK with you?

**Pull request closed without merging**